phys/module_radiation_driver.F

```diff
@@ -1049,9 +1049,15 @@ SUBROUTINE radiation_driver (                                          &
          ENDIF
 ! added coszen subroutine : jararias, 2013/08/10
 !   outputs: coszen, hrang
-         call calc_coszen(ims,ime,jms,jme,its,ite,jts,jte,  &
-                          julian,xtime+radt*0.5,gmt, &
-                          declin,degrad,xlong,xlat,coszen,hrang)
+         if ( solar_opt_local == 1 ) then
```

Comment:
@twjuliano 
Tim,
We probably need a short comment to explain why we have these two different times in otherwise identical subroutine calls based on a flag value.

Reply:
@davegill 
Dave, this has been added.